lib/resolvers.js

```diff
@@ -0,0 +1,97 @@
+class Resolvers {
+  constructor(context) {
+    this.Kuzzle = context.accessors.sdk;
+
+    this._resolversMap = {
+      Query: {
+        allBooks: async () => {
+          const results = await this.Kuzzle.document.search(
+            'test-data',
+            'books'
```

Comment:
In our coding standards, we put the last parenthesis on the same line as the last argument to reduce the total number of lines in a file